apps/laboratory/tests/shared/fixtures/w3m-email-fixture.ts

```diff
@@ -25,3 +28,29 @@ export const testMEmail = timingFixture.extend<ModalFixture>({
     await use(modalValidator)
   }
 })
+
+export const testMEmailSiwe = siwe.extend<ModalFixture>({
+  modalPage: async ({ page, library, context }, use, testInfo) => {
+    const modalPage = new ModalWalletPage(page, library, 'all')
+    await modalPage.load()
+
+    const mailsacApiKey = process.env['MAILSAC_API_KEY']
+    if (!mailsacApiKey) {
+      throw new Error('MAILSAC_API_KEY is not set')
+    }
+    const email = new Email(mailsacApiKey)
+    const tempEmail = email.getEmailAddressToUse(testInfo.parallelIndex)
+
+    await modalPage.emailFlow(tempEmail, context, mailsacApiKey)
+    await modalPage.page.waitForTimeout(1000)
```

Comment:
Same here, instead of putting timeouts we should wait for some elements to be visible. There should be a certain reason when we use timeouts and would better be commented IMO.